sandy/core/endf6.py

```diff
@@ -1805,33 +1816,223 @@ def get_errorr(self,
         groupr
         -21 -22 0 -23 /
         125 1 1 2 0 1 1 0 /
-        /
+        'sandy runs groupr' /
+        0.0/
+        10000000000.0/
+        12 /
+        1.00000e-05 3.00000e-02 5.80000e-02 1.40000e-01 2.80000e-01 3.50000e-01 6.25000e-01 4.00000e+00 4.80520e+01 5.53000e+03 8.21000e+05 2.23100e+06 1.00000e+07 /
+        12 /
+        1.00000e-05 3.00000e-02 5.80000e-02 1.40000e-01 2.80000e-01 3.50000e-01 6.25000e-01 4.00000e+00 4.80520e+01 5.53000e+03 8.21000e+05 2.23100e+06 1.00000e+07 /
+        3/
+        0/
+        0/
+        errorr
+        -21 0 -23 33 0 /
+        125 1 2 0 1 /
+        0 0.0 /
+        0 33 1/
+        12 /
+        1.00000e-05 3.00000e-02 5.80000e-02 1.40000e-01 2.80000e-01 3.50000e-01 6.25000e-01 4.00000e+00 4.80520e+01 5.53000e+03 8.21000e+05 2.23100e+06 1.00000e+07 /
+        stop
+
+        U-238 test:
```

Comment:
Why do we need to test U238? is there something special? if it is because it contains all mubar, xs, chi and nuba, then write it